### ISSUE TITLE
Visual/a11y indication of unpublished content

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#1137]
Follow-up to #985 

## Problem
Unpublished content can appear in views and entity reference fields to logged in users, but then not show to anonymous.

We previously removed styling for Drupal core's unpublished class in uiowa_bootstrap because it interfered with our style background colors and background section images.
https://github.com/uiowa/uiowa/blob/master/docroot/themes/custom/uiowa_bootstrap/scss/partials/02_elements/_node.scss

The pink, as-is, is not WCAG compliant. Also:
>even if you can perceive the pink, you may have no idea what it means

## Proposed solution
We can continue to use the pink color, but we should supplement to cover a11y and content z-indexes. `node.isPublished()` is available for twig logic and it was recommended that a simple text label on the content would cover it.

**Comment by joewhitsitt** [comment]
Additionally blocked. Resources towards uiowa_bootstrap should be avoided at this time.
